get-config.js

```diff
@@ -2,35 +2,28 @@ const fs = require('fs');
 const path = require('path');
 const {npmGlobalModules, yarnGlobalModules, supportGlobalResolving} = require('./global-resolver');
 
-const globalModulesPath = npmGlobalModules() || yarnGlobalModules() || '';
+const globalPaths = [npmGlobalModules(), yarnGlobalModules()].filter(Boolean);
```

Comment:
This seems a little bit sketchy to pull global modules from yarn, just because they might exist, when running on npm (and, vice-versa).  Can we detect which we were launched with?  We can sniff `$npm_execpath` from the environment, it seems, and have a chance to detect which it is.